### ISSUE TITLE
Undefined Font "UiBold"

### ISSUE THREAD

**Issue by Hazbelll** [FPtje/FSpectate#2]
The following Lua error gets spammed to console when spectating a player:
```
[FSpectate] lua/includes/modules/draw.lua:243: 'UiBold' isn't a valid font

  1. SetFont - [C]:-1
   2. WordBox - lua/includes/modules/draw.lua:243
    3. fn - lua/fspectate/cl_init.lua:295
     4. unknown - addons/ulib/lua/ulib/shared/hook.lua:110
```
As the font used for FSpectate's HUD isn't defined by default in other gamemodes like it is in DarkRP, such as Sandbox or Cinema.

https://github.com/FPtje/DarkRP/blob/e19e0066dc666f20327945f2f4392436f1807006/gamemode/modules/base/cl_fonts.lua#L52

**Comment by FPtje** [comment]
Waiting for you to confirm fix before uploading to workshop

**Comment by Hazbelll** [comment]
Can confirm it's fixed.

**Comment by FPtje** [comment]
Uploaded to workshop

**Comment by Hazbelll** [comment]
Actually, I've made a big blunder. The function you defined the font in ends prematurely if FAdmin isn't on the Server, so the font still isn't defined.

https://github.com/FPtje/FSpectate/blob/96435b52475f9926865c9683e499a809830e4399/lua/fspectate/cl_init.lua#L14

Simply need to move it to a line above it and then it'll be sorted. I do apologise, I verified that defining the font fixed it in-game manually and didn't notice that if statement there when I checked the commit.
